modules/cli/src/main/java/org/apache/ignite/internal/cli/core/exception/handler/SslExceptionHandler.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.cli.core.exception.handler;
+
+import javax.net.ssl.SSLException;
+import org.apache.ignite.internal.cli.core.exception.ExceptionHandler;
+import org.apache.ignite.internal.cli.core.exception.ExceptionWriter;
+
+public class SslExceptionHandler implements ExceptionHandler<SSLException> {
+    @Override
+    public int handle(ExceptionWriter err, SSLException e) {
+        err.write("Could not connect to Ignite server. Check SSL configuration.");
```

Comment:
Could you please use the same API that other ExceptionHandlers use? MessageUiCompontes and others.

Also, here the details of what is wrong might be useful.